piptools/utils.py

```diff
@@ -48,13 +48,33 @@ def make_install_requirement(name, version, extras, constraint=False):
     )
 
 
+def is_url_requirement(ireq, repository=None):
+    """
+    Finds if a requirement is a URL
```

Comment:
Just replying quickly to a style issue before I get to the hard stuff. :)

Both are grammatically correct (because it's fine to pronounce it "a you are ell" or "an earl") but "[a URL](https://www.google.com/search?q="a+url")" is about five times as common as "[an URL](https://www.google.com/search?q="an+url")". Could I suggest leaving it as "a URL" to be more likely to match the usage of future PRs?

Reply:
Aw, okay :) Leave it as "a URL", thanks for the info.